app/models/prior_authority_application.rb

```diff
@@ -50,7 +42,7 @@ class PriorAuthorityApplication < ApplicationRecord
   scope :reviewed, -> { where(status: %i[granted part_grant rejected auto_grant sent_back expired]) }
   scope :submitted_or_resubmitted, -> { where(status: %i[submitted provider_updated]) }
 
-  scope :for, ->(provider) { where(office_code: provider.selected_office_code) }
+  scope :for, ->(provider) { where(office_code: provider.office_codes).or(where(provider:)) }
```

Comment:
not 100% sure on this - if a provider has an office code removed from their list then they should no longer be able to see details for that office code - surely???

at least that is my assumption?

Reply:
Ok, so re this bit

> those submission are only visible to the creator until the office code if selected.

Aimee has confirmed that she's OK with that for now.

Re this bit:

> if a provider has an office code removed from their list then they should no longer be able to see details for that office code

What I've done is weaken this clause so that the second half of the 'or' only applies if the application doesn't have a selected office code.